processing/src/test/java/io/druid/query/aggregation/AggregationTestHelper.java

```diff
@@ -591,5 +593,31 @@ public ObjectMapper getObjectMapper()
   {
     return mapper;
   }
+
+  public <T> T[] runRelocateVerificationTest(
+      AggregatorFactory factory,
+      ColumnSelectorFactory selector,
+      Class<T> clazz
+  )
+  {
+    T[] results = (T[]) Array.newInstance(clazz, 2);
+    BufferAggregator agg = factory.factorizeBuffered(selector);
+    ByteBuffer myBuf = ByteBuffer.allocate(10040902);
```

Comment:
The numbers used in this method seem to be something meaningful. Would you explain if they are?